packages/main/src/TableRow.hbs

```diff
@@ -13,12 +13,15 @@
 	aria-selected="{{this.selected}}"
 	data-sap-focus-ref
 	part="row"
+	role="row"
+	navigated="{{navigated}}"
+	dir="{{effectiveDir}}"
```

Comment:
Hi there,
thank you for the PR.

I would like to ask if you need the `dir="{{effectiveDir}}"` attribute set? 
Because, we now try to leverage the new [CSS logical props](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Logical_Properties) and avoid writing CSS styles for LTR or RTL, and let the browsers handle it out of the box.
I don't see any RTL related styles in the change, so maybe it's not needed at all.

Thanks,
ilhan

Reply:
Hi,

Indeed the attribute set is not needed. I removed it. 

Thank you,
Zornitsa.